src/CarsModel.js

```diff
@@ -0,0 +1,47 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import RULES from "./constants/rule.js";
+
+class CarsModel {
+  #carsMap;
+
+  constructor(input) {
+    const cars = input.split(",");
+    this.#carsMap = new Map(cars.map((car) => [car.trim(), 0]));
+  }
+
+  getCarNames() {
+    return [...this.#carsMap.keys()];
+  }
+
+  getCarsMap() {
+    return this.#carsMap;
+  }
+
+  #isCarMove() {
+    const randomNum = MissionUtils.Random.pickNumberInRange(
+      RULES.RANDOM_MIN,
```

Comment:
```suggestion
      RULES.MIN_RANDOM_NUMBER,
```
전체적인 코드 맥락을 보면 `RANDOM_MIN`이 어떤 의미인지 알 수 있을 것 같은데 단독으로 보면 유추하기 어려울 것 같아서 `MIN_RANDOM_NUMBER` 같이 사용하는 것도 괜찮을 것 같아요~!

Reply:
제 커밋 내역 보시면 기존엔 `RANDOM_NUM_MIN`으로 했었습니다 ㅎㅎ 근데 넘 길어서 `RANDOM_MIN`으로 수정했어요. 이렇게만 해도 의도가 충분히 전달된다고 생각했거든요!

[이전 커밋](https://github.com/woowacourse-precourse/javascript-racingcar-7/pull/193/commits/8b7b19387404c53c9d868561d1d6a1767dd5853e#diff-aa79cc3d5db109c8547ef97f1925a1b3dfe51a5e89e7ce58270aa490d7d2a763)

🧐 근데 정말 단독으로 보면 유추하기 어렵긴 하겠네요... 좋은 의견 감사합니다! 👍